scripts/uninstall_istio_system.sh

```diff
@@ -2,10 +2,11 @@
 echo "setting up kubeconfig"
 ./usr/local/app/scripts/init_kubeconfig.sh
 if test -f "/app/overlay-config.yaml"
-then 
-    echo "Unistalling istio and overlay file"    
+then
+    echo "Unistalling istio and overlay file"
     istioctl manifest generate -f /app/istio-base.yaml -f /app/overlay-config.yaml | kubectl delete -f -
-else 
+else
     echo "Unistalling istio"
     istioctl manifest generate -f /app/istio-base.yaml | kubectl delete -f -
+fi
 kubectl delete namespace istio-system
```

Comment:
@cbron since we are not adding the namespace as part of the chart install, should I remove this from the script?

Reply:
Hmm, good question.

Reply:
Will leave it in for now